selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/RenameOrganizationTest.java

```diff
@@ -74,81 +72,69 @@ public void setUp() throws Exception {
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteById(parentOrganization.getId());
+    for (OrganizationDto organization : testOrganizationServiceClient.getAll())
+      testOrganizationServiceClient.deleteById(organization.getId());
   }
 
   @Test(priority = 1)
   public void testParentOrganizationRename() {
     navigationBar.waitNavigationBar();
-    navigationBar.clickOnMenu(NavigationBar.MenuItem.ORGANIZATIONS);
+    navigationBar.clickOnMenu(ORGANIZATIONS);
     organizationListPage.waitForOrganizationsToolbar();
     organizationListPage.waitForOrganizationsList();
 
+    // Open the parent organization and try to set is not valid name
     organizationListPage.clickOnOrganization(parentOrganization.getName());
-
     organizationPage.waitOrganizationTitle(parentOrganization.getName());
-
-    organizationPage.setOrganizationName("");
+    organizationPage.setOrganizationName(" ");
```

Comment:
Is this only one possible invalid name? What about empty name, and name with special symbols like "#$@!&" ?